scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua

```diff
@@ -19,6 +24,17 @@ end;
 -----------------------------------
 
 function onMobSpawn(mob)
+    mob:AnimationSub(0); -- Mouth closed
+    mob:addStatusEffect(EFFECT_FLEE,100,0,30);
```

Comment:
https://github.com/DarkstarProject/darkstar/blob/master/src/map/lua/lua_baseentity.cpp#L4403

just takes icon as a param after id (instead of power being the next param), the rest are the same as addStatusEffect